### PR TITLE
Adds static help && FAQ page

### PR DESCRIPTION
**Pull request by stenington**
@ZeeJab You can build on this for the FAQ/About pages.

Run the app and navigate to `http://localhost:3000/page/yo` and you should see a poorly formatted page that says "Yo." on it. The idea is that `/page/<name>` will render `/clientapp/templates/pages/static/<name>.html` if it finds it, or `404.html` under the same directory if it can't.

So you could make the faq and about pages as templates in that directory and link to them that way. It would also be awesome if you made the 404 page not just say "404." in the same ugly way as the "Yo." example. The only restriction on html templates is that at the top level they **must have a single element**, so wrap everything in a `<section>` or `<div>` or whatever.

### CODE REVIEW

**Comment by threeqube** [comment]
We have a pretty 404 for BadgeKit that we can probably re-purpose: https://github.com/mozilla/openbadges-badgekit/issues/38#issuecomment-41196299

**Comment by ZeeJab** [comment]
The help and FAQ are donezo. Are we using badgekit 404 ^? If so I'll need to add it. @chloeatplay @zzzndr @stenington

**Comment by chloeatplay** [comment]
sweet! @ZeeJab where are you pulling text from? and +1 on reusing the badgekit 404

File: clientapp/templates/pages/static/yo.html [hunk 1]
```diff
@@ -0,0 +1,3 @@
+<div>
+  Yo.
+</div>
```
Comment (stenington) [inline clientapp/templates/pages/static/yo.html:3]: The yo page is haunting you! :ghost:

Looks like this came back in the rebase somehow.

**Comment by stenington** [comment]
The yo page isn't really hurting anything, so...

File: clientapp/templates/pages/static/yo.html (hunk 1 above)
Comment (ZeeJab) [inline clientapp/templates/pages/static/yo.html:3]: noooooo! I will get you next time [yo]. :boat:

**Pull request merged**